Sources/Error.swift

```diff
@@ -1,8 +1,8 @@
-public protocol Error: ErrorProtocol {
+public protocol S4Error: Error {
```

Comment:
I think this should be `Error : Swift.Error`. Then to disambiguate call `S4.Error`.